### ISSUE TITLE
imrelp leaks connections in rsyslog 20.06

### ISSUE THREAD

**Issue by radu-gheorghe** [rsyslog/rsyslog#4350]
This is a spin-off (duplicate?) of #4005, as I can't comment there. We were experiencing the same issue (rsyslog leaking RELP connections). I was expecting the issue to be fixed in rsyslog 20.06, but we see the same issue after the upgrade.

Does this have to do with the fact that the bug was actually (mostly?) in librelp? And it's fixed there, but a new librelp wasn't released? I see the fix (https://github.com/rsyslog/librelp/pull/189) is set for 1.7.0, but I have:
```
# apt list --installed 2>/dev/null | grep relp
librelp-dev/xenial,now 1.6.0-0adiscon1xenial1 amd64 [installed]
librelp0/xenial,now 1.6.0-0adiscon1xenial1 amd64 [installed,automatic]
rsyslog-relp/xenial,now 8.2006.0-0adiscon1xenial1 amd64 [installed]
```

Should I just compile librelp from master? Or is there a librelp "development" package? Or should we wait for librelp 1.7.0?

Please let me know if I can help with anything in this process.

**Comment by rgerhards** [comment]
@radu-gheorghe sry for the silence, it's exceptionally days... I'll see that a co-release librelp 1.7 with rsyslog 8.2008.0

**Comment by radu-gheorghe** [comment]
No worries whatsoever. A co-release would be awesome, I assume an `apt-get upgrade` would fix the problem then :)

**Comment by rgerhards** [comment]
> I assume an `apt-get upgrade` would fix the problem then :)

that's the plan :-)

**Comment by rgerhards** [comment]
closing this as librelp 1.7.0 is already in the release process and will be released together with rsyslog scheduled stable 8.2008.
